packages/wrangler/src/git-client.ts

```diff
@@ -125,8 +127,24 @@ export async function cloneIntoDirectory(
 	// cleanup: move the template to the target directory and delete `.git`
 	try {
 		fs.renameSync(templatePath, targetDirectory);
-	} catch {
-		throw new Error(`Failed to find "${subdirectory}" in ${remote}`);
+	} catch (err) {
+		// @ts-expect-error non standard property on Error
+		if (err.code !== "EXDEV") {
+			logger.debug(err);
+			throw new Error(`Failed to find "${subdirectory}" in ${remote}`);
```

Comment:
Perhaps include the original `err` in the new Error?

Reply:
I'm happy to do this - the original code doesn't, so I assumed that was intentional, but if you think it's safe to just pass it through, can do!

Reply:
Hmm. Oh I see. OK let's leave it as-is.
(Aside, I note that `subdirectory` could actually be `undefined` that would be a weird message!)

Reply:
> (Aside, I note that subdirectory could actually be undefined that would be a weird message!)

Yeah, it could for sure. The error handling here is pretty bare-bones. If you think it's worth refactoring here, let me know.